server/services/meilisearch/client.js

```diff
@@ -8,4 +9,10 @@ const { MeiliSearch: Meilisearch } = require('meilisearch')
  *
  * @returns { object } - Meilisearch client instance.
  */
-module.exports = config => new Meilisearch(config)
+module.exports = config =>
+  new Meilisearch({
+    ...config,
+    clientAgents: [
+      `Meilisearch strapi-plugin-meilisearch ${packageJson.version}`,
```

Comment:
@bidoubiwa I suggest you follow: https://github.com/meilisearch/integration-guides/issues/150#issuecomment-1043262254 because Guillaume amplitude graphs can be impacted!